### ISSUE TITLE
Rename Untitled Diagram.drawio to Ablauf.drawio

### ISSUE THREAD

**Issue by ann0see** [herr-schaefer/iag2020#2]
Der Name des Diagramms ist nicht wirklich aussagekräftig. Was ist die Aussage des Diagramms? Ein Ablauf vor Ort bei den Wettkämpfen?

**Comment by herr-schaefer** [comment]
ja, das war ein klassischer anfängerfehler. ehrlich gesagt ist es auch noch nicht fertig ;-)
Ist doch komplexer als ich zunächst dachte. Vielleicht entwerfe ich es erstmal mit papier und stift.

**Comment by ann0see** [comment]
Ok. Dann wird das sicher noch nächste Woche besprochen.
